hack/upstream-install.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+ROOT_DIR=$(dirname "${BASH_SOURCE}")/..
+source "${ROOT_DIR}/hack/common.sh"
+source "${ROOT_DIR}/hack/lib/customize-manifests.sh"
+
+set +u
+
+if [[ -z "$DISABLE_OCP_FEATURES" || "$DISABLE_OCP_FEATURES" = "false" ]]; then
```

Comment:
This can most likely be reduced into ```INSTALLER_MANIFESTS_DIR="$UPSTREAM_INSTALLER_MANIFESTS_DIR"``` throughout the upstream install/uninstall scripts.

Reply:
Yes there shouldn't be any use of `DISABLE_OCP_FEATURES` in here. And yes, we'll want to set `export INSTALLER_MANIFESTS_DIR="$UPSTREAM_INSTALLER_MANIFESTS_DIR"`